src/components/ButtonGroup/ButtonGroup.jsx

```diff
@@ -128,7 +132,7 @@ ButtonGroup.sizes = Button.sizes;
 ButtonGroup.kinds = Button.kinds;
 
 ButtonGroup.defaultProps = {
-  componentClassName: "",
+  className: undefined,
```

Comment:
Don't think it's needed.

Reply:
That reason for this is because if the  class name will contain default value we will ignore the value of the deprecated componentClassName.

Reply:
I can set a default value check for the backward compatibility function but I think it's not really matter (this way or another)